launcher/control.py

```diff
@@ -267,19 +246,16 @@ def init(self):
                 "name": project["name"],
             }, **project["data"])
             for project in sorted(io.projects(), key=lambda x: x['name'])
-
-            # Discard hidden projects
-            if project["data"].get("visible", True)
+            if project["data"].get("visible", True)  # Discard hidden projects
         ])
 
-        # No actions outside of projects
-        self._actions.push([])
-
-        frame = {
-            "environment": {},
-        }
+        frame = {"environment": {}}
```

Comment:
The "cosmetic" changes like this one and the one above aren't necessarily more readable and are both (I think) valid PEP08. Of course, it's a personal subjective feeling.

To me it's not better or worse, just wanted to chime in that the changes are somewhat personal and might be redundant if the code's already formatted a way by someone else. Unless code style requires this changes, then... nothing said. :)